language/compiler/ir-to-bytecode/syntax/src/syntax.rs

```diff
@@ -1464,6 +1464,26 @@ fn parse_unary_spec_exp<'input>(
             consume_token(tokens, Tok::RParen)?;
             SpecExp::Old(Box::new(exp))
         }
+        Tok::NameValue => {
+            let next = tokens.lookahead();
+            if !next.is_ok() || next.unwrap() != Tok::LParen {
```

Comment:
Minor comment:  If you swapped the then and else branch the condition would become 

if next.is_ok() && next.unwrap() == Tok::LParen { ... } else { ... }

which I find much easier to read.

Reply:
Removed the !ok, but kept the order. There is a general style rule to make the shorter or default treatment at the beginning not at the end of an if-then-else.